rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
```

Comment:
Again, this rule always returns true, even when popup is closed. I'd suggest `{ "visible": "sibbo-cmp-layout" }`